examples/wcmp2_de.dwd.icon-eps.ALL.json

```diff
@@ -20,126 +20,111 @@
-                    }
+                    {"id": "Meteorological geographical features"}
                 ],
                 "scheme": "https://inspire.ec.europa.eu/theme/mf"
             },
             {
                 "concepts": [
-                    {
-                        "id": "FM 92 GRIB edition 2"
-                    }
+                    {"id": "FM 92 GRIB edition 2"}
                 ],
                 "scheme": "http://codes.wmo.int/codeform"
-            }
+            },
+			{
+				"concepts": [
+					{"id": "continual"}
+				],
+				"scheme": "https://standards.iso.org/iso/19139/resources/gmxCodelists.xml#MD_FrequencyCode"
+			}
         ],
-        "providers": [
+		"contacts": [
             {
-                "name": "Deutscher Wetterdienst",
+                "identifier": "DWD",
+				"organization": "Deutscher Wetterdienst",
                 "positionName": "Michael Denhard",
-                "contactInfo": {
-                    "email": {
-                        "work": "gisc@dwd.de"
-                    },
-                    "address": [
+				"emails": [{
+                        "value": "wis@dwd.de"
+                    }],
+                "addresses": [
                         {
-                            "deliveryPoint": "Frankfurter Stra\u00dfe 135",
+                            "deliveryPoint": ["Frankfurter Strasse 135"],
                             "city": "Offenbach",
-                            "postal-code": "63067",
+                            "postalCode": "63067",
                             "country": "Germany"
                         }
                     ],
-                    "url": "http://www.dwd.de/wmc"
-                },
-                "roles": [
-                    {
-                        "name": "pointOfContact"
-                    },
-                    {
-                        "name": "originator"
-                    }
-                ]
+				"links": [
+					{
+						"href": "https://www.dwd.de/wmc"
+					},
+					{
+						"href": "https://gisc.dwd.de"
+					}
+				],
+				"contactInstructions": "email",
+                "roles": ["pointOfContact","originator"]
             }
         ],
         "created": "2018-08-19T08:10:00Z",
         "updated": "2022-03-15T09:26:00Z",
-        "wmo:dataPolicy": "core",
-        "links": [
-            {
-                "rel": "download",
-                "type": "text/html",
-                "title": "Open Data Server DWD",
-                "href": "https://opendata.dwd.de/weather/wmc/icon-eps/data/grib"
-            },
-            {
-                "rel": "items",
-                "type": "application/json",
-                "title": "Message Broker DWD",
-                "href": "mqtts://everyone:everyone@wis2.dwd.de",
-                "channel": "origin/a/wis2/deu/dwd-offenbach/data/core/weather/analysis-prediction/forecast/model/#"
-            }
-        ],
+        "wmo:dataPolicy": {
```

Comment:
Should be `"wmo:dataPolicy": "core"`

Reply:
I knew I forgot something... changed - thanks!